sql/core/src/main/scala/org/apache/spark/sql/execution/limit.scala

```diff
@@ -358,7 +358,8 @@ case class TakeOrderedAndProjectExec(
     val orderByString = truncatedString(sortOrder, "[", ",", "]", maxFields)
     val outputString = truncatedString(output, "[", ",", "]", maxFields)
 
-    s"TakeOrderedAndProject(limit=$limit, orderBy=$orderByString, output=$outputString)"
+    val offsetStr = if (offset == 0) { "" } else {  s" offset=$offset,"  }
```

Comment:
How about `val offsetStr = if (offset > 0) s" offset=$offset," else ""` ?

Reply:
> How about `val offsetStr = if (offset > 0) s" offset=$offset," else ""` ?

@beliefer Thank you for your review.  I made the modifications according to what you said, but there are some UTs that cannot run through. Can you help me take a look.

Reply:
It seems the GA failure is unrelated.

Reply:
It seems the GA failure is unrelated.

Reply:
> It seems the GA failure is unrelated.

@beliefer  OK. Thank you.